avm/res/api-management/service/tests/e2e/consumptionSku/main.test.bicep

```diff
@@ -0,0 +1,54 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults with a Consumption SKU'
```

Comment:
Do we want to show the usage of a Consumption sku or in particular a Consumption sku defaults?

In the former case, please update title and description to not mention defaults.
In the latter case, please remove any parameter which is not of type Required.